internal/ir/xds.go

```diff
@@ -175,6 +179,39 @@ type RouteDestination struct {
 	Weight uint32
 }
 
+// +k8s:deepcopy-gen=true
+type DirectResponse struct {
+	// Body can actually support a config.core.v3.DataSource, but for now only an
+	// inline string is supported
```

Comment:
Nit: These are complete sentences, so they should have punctuation. (I wouldn't bother with this comment on a plain comment in a function or something, but these are doc comments.)